Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -3125,12 +3127,35 @@ def splunk_search_command(service: client.Service, args: dict) -> CommandResults
     return results
 
 
-def splunk_job_create_command(service: client.Service, args: dict):
+def splunk_job_create_command(service: client.Service, base_url: str, token: str, auth_token: str | None, args: dict):
     app = args.get("app", "")
+    share = args.get("share", "No")
```

Comment:
i dont see any usage of the arg share right after defining it, so why did you add it?